docs/topics/api-references.topic

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic SYSTEM "https://resources.jetbrains.com/writerside/1.0/xhtml-entities.dtd">
+<topic xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
+       id="api-references" title="API references">
+
+    <p>Welcome to the Kotlin API References page. Here you'll find links to the API documentation for the official Kotlin libraries and tools.</p>
+
+    <list columns="2">
+        <li>
+            <a href="https://kotlinlang.org/api/core/kotlin-stdlib/"><b>Standard library (stdlib)</b></a>
+            <br/>
+            <p>The Kotlin Standard library provides core functionality for Kotlin programming. It includes essential APIs such as collections, text and string processing, I/O, ranges, sequences, and more, and is available on all supported platforms.</p>
```

Comment:
I/O-specific API is not available on all platforms, it's JVM-specific.

Perhaps, we can exclude it from this list, and also add that the standard library extends platform-specific APIs and provides a Kotllin-first API to work with it.

CC @ilya-g